package/batocera/utils/rpigpioswitch/S92switch

```diff
@@ -4,18 +4,35 @@ NAME="S92switch"
 RUN="/usr/bin/rpi_gpioswitch"
 BTD_PID=$(ps -eo pid,args | grep "/bin/bash $RUN" | grep -v grep | awk '{print $1}')
 
+#Add new devices here and modify /usr/bin/rpi_gpioswitch, too
+devices=(
+         ATX_RASPI_R2_6
+         REMOTEPIBOARD_2003
+         REMOTEPIBOARD_2005
+         MAUSBERRY
+         WITTYPI
+         ONOFFSHIM
+         RETROFLAG
+         PIN56ONOFF
+         PIN56PUSH
+         PIN356ONOFFRESET
+        )
+
 # Carry out specific functions when asked to by the system
 case "$1" in
     start)
-	systemsetting="python /usr/lib/python2.7/site-packages/configgen/settings/batoceraSettings.py"
-	powerswitch="`$systemsetting  -command load -key system.power.switch`"
-	case "${powerswitch}" in
-	    ATX_RASPI_R2_6|MAUSBERRY|REMOTEPIBOARD_2003|REMOTEPIBOARD_2005|WITTYPI|PIN56ONOFF|PIN56PUSH|PIN356ONOFFRESET|ONOFFSHIM)
-		nice -n 19 $RUN start $powerswitch &
-		;;
-	esac
+        systemsetting="python /usr/lib/python2.7/site-packages/configgen/settings/batoceraSettings.py"
+        powerswitch="`$systemsetting  -command load -key system.power.switch`"
+        for i in ${devices[@]}; do
+            [ "$i" == "$powerswitch" ] && break
+        done
+        if [ $? -eq 0 ]; then
+            nice -n 19 $RUN start $powerswitch &
```

Comment:
Do you need to "nice -n 19" it here?

Reply:
It's original code - I think it does not matter to change but will not hurt.